options.lua

```diff
@@ -112,6 +113,12 @@ interfaceOptions = {
 					desc = "Use Soulstone/Ankh/etc. resurrection option when one is available and a normal/battle resurrection is not\n\nThis option |cffff0000ignores|r the |cffffd100Don't Accept Revives|r option!",
 					descStyle = "inline", width = "full", type = "toggle", set = optionSetter, get = optionGetter,
 				};
+				dontPostAuctions = {
+					order = 10,
```

Comment:
I'd probably move this up to 4 and shift everything else down.

I did the manual numbering out of habit, but it might not actually be necessary.